cave/plot/configurator_footprint.py

```diff
@@ -355,131 +371,174 @@ def get_conf_matrix(self):
         return np.array(conf_matrix), conf_list, runs_runs_conf
 
     def _get_size(self, r_p_c):
-        return 10 + ((r_p_c - self.min_runs_per_conf) / (self.max_runs_per_conf - self.min_runs_per_conf)) * 40
-
-    def plot(self, X, conf_list: list, runs_runs_conf, inc_list: list,
-             runs_labels=None, contour_data=None):
-        '''
-            plots sampled configuration in 2d-space;
-            saves results in self.output, if set
+        sizes = 10 + ((r_p_c - self.min_runs_per_conf) / (self.max_runs_per_conf - self.min_runs_per_conf)) * 40
```

Comment:
Bokeh seems to handle marker sizes differently.
So we might have to scale back a bit.

Reply:
At least to me the final incumbent looks unproportionally large